tests/test-cases/Group2-Docker-Archive/2-01-Docker-Archive.robot

```diff
@@ -0,0 +1,114 @@
+# Copyright 2016-2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation  Test 2-01 - Docker Archive
+Resource  ../../resources/Util.robot
+Suite Setup  Install VIC Appliance To Test Server  additional-args=--debug=3
```

Comment:
We are already passing --debug 1 as the default level during all tests, not sure how this works (passing debug level 2x).  We should probably re-work Install VIC Appliance To Test Server to allow different debug levels.